features/nfc/source/nfc/NFCEEPROM.cpp

```diff
@@ -16,12 +16,13 @@
 
 #include "NFCEEPROM.h"
 #include "ndef/ndef.h"
+#include "Event.h"
```

Comment:
redundant, already included in header

Reply:
pushed a new commit to remove the line - compiles fine